ParkingZoneApp/Areas/Admin/Views/ParkingZones/Delete.cshtml

```diff
@@ -0,0 +1,48 @@
+﻿@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@model ParkingZoneApp.Areas.Admin.Models.ParkingZone
+
+@{
+    ViewData["Title"] = "Delete";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+
```

Comment:
remove this line

Reply:
ok  but this line remaining @{Layout = "~/Views/Shared/_Layout.cshtml";}